[1LP][RFR] NewTest:: testing network configuration using negative input

dgaikwad opened the pull request:
<!-- Make sure to prefix the PR Title with any one of the below options...

   [WIP] - Work in Progress
   [WIPTEST] - Work in Progress with PRT Testing
   [RFR] - Ready for 1st Review
   [1LP][WIP] - 1st Level Pass, Work in Progress
   [1LP][WIPTEST] - 1st Level Pass, Work in Progress with PRT Testing
   [1LP][RFR] - 1st Level Pass, Ready for 2nd Review
-->

## Purpose or Intent
<!-- Please provide some information related to PR. Some examples are:

- __Fixing__ X because it is currently doing Y which is incorrect. It should be doing Z.
- __Extending__ X because I need it to do Y so that I can update/add more test cases; PR #123
depends on it.
- __Adding tests__ for feature X to cover cases Y and Z. They were implemented in product
version1.2.3.
- __Updating tests__ for feature X because the behavior changed in product version 1.2.3.
- __Enhancement__ for feature X

Note: You can introduce Screenshots/Gifs for providing more information.
-->

### PRT Run
- Testing network configuration using negative input
Testing below options:
1) Set DHCP Network Configuration
4) Test Network Configuration
5) Set Hostname

<!--
- It's an internal RH service and only runs against signed whitelisted commits.
- It runs against a single appliance.
- Need to provide specific pytest expression else default it will run smoke tests [-m smoke].
- DOUBLE CURLY BRACES REQUIRED. Examples are in single to not get picked

Some examples are:
- {pytest: cfme/tests/test_foo_file.py -v}
- {pytest: cfme/tests/test_foo_file.py -k "test_foo_1 or test_foo_2" -v}
- {pytest: cfme/tests/ -k "test_foo_1 or test_foo_2 or test_foo_3" -v}
- {pytest: cfme/tests/test_foo_file.py --use-provider rhv43 -v}
- {pytest: cfme/tests/test_foo_file.py --long-running -v}

Notes:
- If PRT fails other than PR purpose/intent; please add a respective comment.
- For any guidance or assistance with PRT results; please contact to maintainers.
-->
{{pytest: cfme/tests/cli/test_appliance_console.py::test_appliance_console_network_conf_negative --long-running -v}}

ganeshhubale commented on cfme/tests/cli/test_appliance_console.py:
```diff
@@ -1007,7 +1011,42 @@ def test_appliance_console_network_conf_negative():
+    assert new_data.success
+    new_data = new_data.output.split("\n")
+    logger.info("\'ap\' command's output after test:{}".format(new_data))
+
```
same here for fstring

digitronik commented on cfme/tests/cli/test_appliance_console.py:
```diff
@@ -994,20 +1022,59 @@ def test_appliance_console_network_conf_negative():
+        new_config_file = fauxfactory.gen_alphanumeric(start="test_", length=10)
+        command_set = (f"ap | tee -a /tmp/{new_config_file}", RETURN)
+        appliance.appliance_console.run_commands(command_set, timeout=30)
+        command_set = (f"cat /tmp/{new_config_file}")
```
Why we need to create extra file with data and then cat it?
`run_commands` output arg not helpful here?

dgaikwad commented:
Following action performed after cove review:
- Removed parseIPV6() and parseIPV4() functions, this function will be implemented as you suggested and will use same where we need.
- Improved formatting
- updated logic where I was storing "ap" command's output in the text file

dajoRH commented:
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*
